iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -51,20 +77,36 @@ public enum ErrorCode
         //[EditorBrowsable(EditorBrowsableState.Never)]
         BulkRegistryOperationFailure = 400013,
 
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be returned by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
         CannotRegisterModuleToModule = 400301,
 
         // Unauthorized - 401
         IotHubNotFound = 401001,
 
-        IotHubUnauthorizedAccess = 401002,
-
         /// <summary>
         /// The SAS token has expired or IoT hub couldn't authenticate the authentication header, rule, or key.
         /// For detailed information, see <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-troubleshoot-error-401003-iothubunauthorized"/>.
         /// </summary>
-        IotHubUnauthorized = 401003,
+        IotHubUnauthorizedAccess = 401002,
+
+        /// <summary>
+        /// Unused error code. Service does not return it and neither does the SDK.
+        /// </summary>
+        [Obsolete("This error does not appear to be thrown by the service.")]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        IotHubUnauthorized = 401003, // never thrown by service anymore. Replaced by IotHubUnauthorizedAccess
```

Comment:
Want to move the code comment to remarks or something?

Reply:
Sure thing